<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Авторизация</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QWidget" name="centerWidget" native="true">
     <layout class="QVBoxLayout" name="centerLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="logoLabel">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="pixmap">
         <pixmap>images/images/logo.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="authLabel">
        <property name="styleSheet">
         <string>
          font-size: 24px;
          font-weight: bold;
          color: #2E2E2E;
         </string>
        </property>
        <property name="text">
         <string>Авторизация</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLineEdit" name="emailInput">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string>
          border: 2px solid #CCCCCC;
          border-radius: 10px;
          padding: 8px;
          font-size: 14px;
         </string>
        </property>
        <property name="placeholderText">
         <string>Почта</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLineEdit" name="passwordInput">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string>
          border: 2px solid #CCCCCC;
          border-radius: 10px;
          padding: 8px;
          font-size: 14px;
         </string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Пароль</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="loginButton">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string>
          background-color: #4800D4;
          color: #FFFFFF;
          font-size: 16px;
          border-radius: 10px;
          padding: 10px;
         </string>
        </property>
        <property name="text">
         <string>Войти</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
